src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSet.java

```diff
@@ -4912,6 +4912,23 @@ final boolean hasUpdatedColumns() {
         return false;
     }
 
+    private String getUpdatedColumnTableName() throws SQLServerException {
+        String columnTableName = "";
+
+        for (Column column : columns) {
+            if (column.hasUpdates() && columnTableName.isEmpty()) {
+                columnTableName = column.getTableName().asEscapedString();
+            } else if (column.hasUpdates() && !columnTableName.isEmpty()
+                    && !columnTableName.equals(column.getTableName().asEscapedString())) {
```

Comment:
Is the isEmpty() needed here?
```suggestion
            } else if (column.hasUpdates() && !columnTableName.equals(column.getTableName().asEscapedString())) {
```

Reply:
Yes, if it is not empty it means it was already set prior. If it was set prior, then we need to check if it matches the prior name. If it doesn't match then we throw.